File: Java/Algorithm/Validation2.java
public void decrement_item(View view) {
    if (quantity_item > 0) {
        quantity_item--;
        display_item(quantity_item);
    }
}
Também poderia fazer algo assim para informar melhor:

public void decrementItem(View view) {
    if (quantityItem > 0) {
        quantityItem--;
        displayItem(quantityItem);
    } else {
        messagemErro("Não pode ficar com valor inferior a zero");
    }
}

//https://pt.stackoverflow.com/q/237167/101

File: Python/Algorithm/Validation2.py
def filtro(p):
    if p > 6:
        return True
    return False

x = int(input("Valor de x"))
if filtro(x): print(x)
x = int(input("Valor de x"))
if filtro(x): x = x * 2
print(x)
x = int(input("Valor de x"))
if filtro(x): x = x * 2
print(x)

#https://pt.stackoverflow.com/q/274003/101
